### ISSUE TITLE
can you provide a docker to run?

### ISSUE THREAD

**Issue by anqin** [Shreyas9400/SearchGPT#2]
want to look inside the codes and need to try running the codes within a docker

**Comment by Shreyas9400** [comment]
I have created a requirements.txt file, will upload a docker file soon. You might also have to generate APIs for handling inference and llamaparse. Will soon provide a documentation on it. In the meantime you may test the product here : https://shreyas094-searchgpt.hf.space

**Comment by anqin** [comment]
OR， it would be perfect if there exists a step-by-step commands to run the codes.

**Comment by anqin** [comment]
> I have created a requirements.txt file, will upload a docker file soon. You might also have to generate APIs for handling inference and llamaparse. Will soon provide a documentation on it. In the meantime you may test the product here : https://shreyas094-searchgpt.hf.space

Yes, the website is nice. But I want to know how to run in local
